### DISCUSSION TITLE
relation "tags" does not exist

### DISCUSSION THREAD

**Discussion by PraisonMaik** [akiver/cs-demo-manager#811] [Q&A]
after following the instalation guide, when I try to run the app I get `relation "tags" does not exist` error when trying to connect. what can I do to make it work?

**Comment by samarrero** [comment]
Getting the same thing with a fresh PostgreSQL 1.16.2 install

**Comment by akiver** [comment]
It will be fixed, thanks for the report

**Comment by akiver** [comment]
Fixed as of v3.5.1
